lib/LedgerSMB/Scripts/budgets.pm

```diff
@@ -50,8 +50,8 @@ sub new_budget {
 
 sub _render_screen {
     my ($budget) = @_;
-    my $additional_rows = 5;
-    $additional_rows +=20 unless $budget->lines;
+    my $additional_rows = 5;  ## no critic (ProhibitMagicNumbers) sniff
+    $additional_rows +=20 unless $budget->lines;  ## no critic (ProhibitMagicNumbers) sniff
```

Comment:
5 could be the constant ADDITIONAL_ROWS whereas 20 could be BASE_ROW_COUNT?

Reply:
I recognized the similarity to the check writing code, iirc.   This made me wonder about the correctness of that change.  Why would a budget and  check stub have the same  limits?

Reply:
How about EDIT_BUDGET_ROWS and NEW_BUDGET_ROWS?   It looks like the 25 lines are for an empty budget to be entered and 5 lines are a trailer for expanding a budget.

Reply:
Sounds good. Go for it.